knownprojects_build/_sql/create_corrections.sql

```diff
@@ -0,0 +1,34 @@
+-- first round of corrections -> project to project
+DROP TABLE IF EXISTS corrections_project;
```

Comment:
We still need to be able to run against an empty corrections table to create the pre-review projects. Are we going to do this by keeping an empty csv in `data/corrections/`?

Reply:
can you remind me where is the code for that?

Reply:
you mean like this? 
https://github.com/NYCPlanning/db-knownprojects-data/pull/9

Reply:
That works, too. I just did a "CREATE TABLE IF NOT EXISTS" statement to accomplish the same goal.

Reply:
I see, but we wouldn't want anything to persist across runs, each run should start with the same clean state, otherwise it would start to get confusing